### PR TITLE
[bashible] Forbid using Docker 18.* 

### PR DESCRIPTION
**Pull request by name212**
## Description
Forbid using Docker 18.* 

Allow only 19.* and 20.* versions.

## Why do we need it, and what problem does it solve?
We have troubles on 18.* Docker during release new Deckhouse version.
Docker often stuck and we should reboot node.

## What is the expected result?
On nodes with Docker 18.* bashible should request disruptive update on step `031_install_docker.sh` 

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: bashible
type: feature
summary: Forbid using Docker 18.*
impact: |
  After upgrading Deckhouse all nodes with Docker 18.* will request `disruptive update`. You will receive `NodeRequiresDisruptionApprovalForUpdate` if you have manual `approvalMode` in NodeGroup.
impact_level: high
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @name212. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @name212. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @name212. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @name212. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by name212** [comment]
@konstantin-axenov Please verify impact message too

File: candi/version_map.yml [hunk 1]
```diff
@@ -129,7 +129,7 @@ bashible: &bashible
   centos:
     '7':
       docker:
-        desiredVersion: "docker-ce-18.09.9-3.el7.x86_64"
+        desiredVersion: "docker-ce-19.03.15-3.el7.x86_64"
```
Comment (konstantin-axenov) [inline candi/version_map.yml:132]: Have you checked that this version is available in centos 7 and rhel 7?

Comment (konstantin-axenov) [reply candi/version_map.yml:132]: Discussed. Everything is fine.

**Review by konstantin-axenov — APPROVED** [review]

**Comment by z9r5** [comment]
Cherry-picked into release-1.34.

**Pull request merged**